src/analysis/raster/qgsrastercalculator.cpp

```diff
@@ -544,7 +544,7 @@ QgsRasterCalculator::Result QgsRasterCalculator::processCalculationGPU( std::uni
     programTemplate = programTemplate.replace( QLatin1String( "##EXPRESSION##" ), cExpression );
     programTemplate = programTemplate.replace( QLatin1String( "##EXPRESSION_ORIGINAL##" ), calcNode->toString() );
 
-    //qDebug() << programTemplate;
+    // qDebug() << programTemplate;
```

Comment:
Should we get rid of this altogether or transform it into a proper QGIS debug log with some elevated level value?

Reply:
you can remove it.